receiver/sqlserverreceiver/scraper.go

```diff
@@ -581,7 +582,7 @@ func (s *sqlServerScraperHelper) recordDatabaseQueryTextAndPlan(ctx context.Cont
 
 		s.logger.Debug(fmt.Sprintf("QueryHash: %v, PlanHash: %v, DataRow: %v", queryHashVal, queryPlanHashVal, row))
 
-		record.Attributes().PutInt(dbPrefix+totalElapsedTime, totalElapsedTimeDiffs[i])
+		record.Attributes().PutDouble(dbPrefix+totalElapsedTime, float64(totalElapsedTimeDiffs[i])/1000)
```

Comment:
Nit: Maybe in the future we should try to add units to variable names, or something similar. I saw `elapsedTime` was already being divided by 1000, then divided here again by 1000 and was pretty confused. (I see now that it's recorded in microseconds in the SQL table, but being more clear about units would help going forward)